README.md

```diff
@@ -163,6 +163,13 @@ nix-shell> nixpkgs-review comments
 `nixpkgs-review` will by default use [nix-output-monitor](https://github.com/maralorn/nix-output-monitor) if found in `$PATH`.
 If you have `nom` installed but don't want to use it, you can pass `nix` to `--build-graph` to use `nix build` instead of `nom build`.
 
+Some pull requests may require configuration for nixpkgs to test out. You can
+use the `--extra-nixpkgs-config` flag to supply extra configuration for nixpkgs.
+
+```console
+$ nixpkgs-review pr 37242 --extra-nixpkgs-config '{ config.cudaSupport = true; }'
```

Comment:
We should omit the ``{}`` or enforce/check them otherwise things will break.

Reply:
I don't think we should omit `{}` because the argument would not be valid nix, unless we want something more abstract like `--extra-nixpkgs-config config.cudaSupport true`, the trailing semicolon can be unintuitive to users

maybe we can evaluate it first without nixpkgs so the error messages will be better? enforcing `{}` can have false positives if things builtin functions are used

Reply:
It should be fine to enforce people to supply something like ``{ ... }``. They should not code in this anyway.

Reply:
> use `--extra-nixpkgs-arg` (and `--extra-nixpkgs-argstr`) instead, similar to `nix-env`'s `--arg` and `--argstr`

What do you think about this option? @SuperSandro2000